src/components/Calendar/Toolbar/CustomEventDialog/DaySelector.tsx

```diff
@@ -11,6 +11,12 @@ interface DaySelectorProps {
 interface DaySelectorState {
     days: boolean[];
 }
+
+function detectMobile() {
+    const isMobile = window.innerWidth <= 800 && window.innerHeight <= 600;
+    return isMobile;
+}
+
```

Comment:
Please try to use the [MUI utility](https://v4.mui.com/components/use-media-query/#simple-media-query) for this if you can; using the `window` object is generally unadvisable because it's not necessarily consistent with the state of the app.

[An example of the usage in MUI5 is here](https://github.com/icssc/AntAlmanac/blob/aponia-monorepo/apps/client/src/routes/Home.tsx#L19), where I used the theme's breakpoints in conjunction with the media query. However, the examples are for functional components; you may need to adapt it for this class component.